dev-libs/granite/granite-5.2.4.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION=0.40
+
+inherit meson vala xdg-utils
+
+DESCRIPTION="Elementary OS library that extends GTK+"
+HOMEPAGE="https://github.com/elementary/granite"
+SRC_URI="https://github.com/elementary/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+# FIXME: Figure out issues with enabling doc.
+# See https://github.com/gentoo/gentoo/pull/12690#issuecomment-525027164
+IUSE="test"
+
+BDEPEND="
+	$(vala_depend)
+	>=dev-util/meson-0.48.2
+	virtual/pkgconfig
+"
+DEPEND="
+	>=dev-libs/glib-2.50:2
+	dev-libs/libgee:0.8[introspection]
+	>=x11-libs/gtk+-3.22:3[introspection]
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	vala_src_prepare
+
+	# if use doc; then
+	# 	# Fixes wrong valadoc bin name
+	# 	sed -i \
+	# 	"s/find_program('valadoc')/find_program('valadoc-$(vala_best_api_version)')/g" \
+	# 	doc/meson.build || die "Failed to sed doc/meson.build"
+
+	# 	# Fixes doc file location
+	# 	mkdir "${WORKDIR}/doc" || die
+	# 	cp -r doc/* "${WORKDIR}/doc" || die
+	# fi
+
+	eapply_user
+}
+
+src_configure() {
```

Comment:
You should be able to remove this phase altogether now.

Reply:
Good reminder